src/parser/TraverseSentences.h

```diff
@@ -307,16 +307,18 @@ class FetchVerticesSentence final : public Sentence {
         yieldClause_.reset(clause);
     }
 
-    explicit FetchVerticesSentence(Expression *ref) {
+    explicit FetchVerticesSentence(Expression *ref, YieldClause  *clause) {
```

Comment:
remove explicit

Reply:
This will disable the implicit convert ctor.